app/views/layouts/application.html.erb

```diff
@@ -3,7 +3,11 @@
 <head>
   <title>Chess</title>
   <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
```

Comment:
when i remove the stylesheet one it causes problems and doesnt display the board